app/src/main/java/de/grobox/transportr/trips/search/DirectionsActivity.java

```diff
@@ -147,6 +152,26 @@ private void processIntent() {
 					to = viewModel.getWork().getValue();
 					if (to == null) new WorkPickerFragment().show(getSupportFragmentManager(), WorkPickerFragment.TAG);
 					break;
+				case INTENT_URI_FAVORITE:
+					from = null;
+					via = null;
+					to = null;
+					long uid = intent.getLongExtra("uid",0);
```

Comment:
Missing space. Btw. in Android Studio, there's a shortcut to automatically apply the current coding style to the entire file or your selection. This quickly fixes the style.